server/internal/graphql/resolver/pattern.go

```diff
@@ -24,7 +24,7 @@ func (r *Resolver) fetchPatterns(ctx context.Context, provider models.Provider,
 	if selector.Search != nil {
 		search = *selector.Search
 	}
-	resp, err := provider.GetMesheryPatterns(tokenString, selector.Page, selector.PageSize, search, order, updateAfter)
+	resp, err := provider.GetMesheryPatterns(tokenString, selector.Page, selector.PageSize, search, order, models.Public, updateAfter)
```

Comment:
For this GraphQL resolver, instead of always fetching public content, can you update the GraphQL schema to include this as an input? (similar to Page, PageSize, Search...)